packages/fluentui/react-northstar/test/specs/components/Portal/Portal-test.tsx

```diff
@@ -73,7 +73,7 @@ describe('Portal', () => {
   describe('onMount', () => {
     it('called when portal opens', () => {
       const onMount = jest.fn();
-      const wrapper = mountWithProvider(<Portal content={<p />} onMount={onMount} />);
+      const wrapper = mountWithProvider(<Portal content={<p />} onMount={onMount} open={false} />);
```

Comment:
Our tests where switching from autocontrolled to controlled mode. Fix is needed.